docs/workgroup/workgroup_charter.rst

```diff
@@ -107,27 +107,27 @@ tool.
 Budget
 ------
 
-The working group will not require a dedicated budget in its initial phase.
-If needed, the working group may ask the board of directors for any funds needed
+The Editorial Board will not require a dedicated budget in its initial phase.
+If needed, the Editorial Board may ask the PSF board of directors for any funds needed
 (for meetings, writers, translators, software development, etc.).
 
 Participants
 ------------
-The Python Steering Council are permanent members of this working group.
+The Python Steering Council are permanent members of the Editorial Board.
 The initial workgroup members include:
```

Comment:
One 'workgroup' that hasn't been changed, and a wording suggestion to try and make this more of a current ToR rather than a proposal document:

```suggestion
The current members are:
```